Makefile

```diff
@@ -50,7 +70,9 @@ openapi:
 	echo '}' >>  ./cmd/mailchain/internal/http/handlers/openapi.go
 	addlicense -l apache -c Finobo ./cmd/mailchain/internal/http/handlers/openapi.go	
 	rm -rf vendor
-	
+fmt:
+	go fmt ./...
+
```

Comment:
Please remove - Seems unneeded as the `go fmt ./...` is quite well known already. I'd remove. Plus the issue with `gofmt -s ...` is that it needs to be ran on a per file basis as it updates the file so cant be ran against all directories like this

Reply:
Ok, i will change this,